recipes-devtools/clang/clang/0029-OpenMP-link-fix.patch 

```diff
@@ -0,0 +1,26 @@
+From 11bd98436bd7e99fb1fc2c07c150c71e2250a262 Mon Sep 17 00:00:00 2001
+From: Joel Winarske <joel.winarske@gmail.com>
+Date: Wed, 9 Feb 2022 11:11:55 -0800
+Subject: [PATCH] OpenMP link fix
+
```

Comment:
Can you add `Upstream-Status` and link to patch from llvm bugzilla here.

Reply:
sure

Reply:
Bug transferred here:  https://github.com/llvm/llvm-project/issues/38548

Reply:
Can you also see if we have this issue in newer clang 13+ as well ? maybe try master branch